node/tests/RedisClient.test.ts

```diff
@@ -297,6 +297,67 @@ describe("GlideClient", () => {
         },
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "lolwut test_%p",
+        async (protocol) => {
+            const client1 = await GlideClient.createClient(
+                getClientConfigurationOption(cluster.getAddresses(), protocol),
+            );
+            const client2 = await GlideClient.createClient(
+                getClientConfigurationOption(cluster.getAddresses(), protocol),
+            );
+
+            // test with multi-node route
+            const result1 = await client1.lolwut(undefined, undefined);
+            expect(intoString(result1)).toEqual(
+                expect.stringContaining("Redis ver. "),
+            );
+
+            const result2 = await client1.lolwut(undefined, []);
+            expect(intoString(result2)).toEqual(
+                expect.stringContaining("Redis ver. "),
+            );
+
+            // test with single-node route
+            const result3 = await client1.lolwut(undefined, [50, 20]);
+            expect(intoString(result3)).toEqual(
+                expect.stringContaining("Redis ver. "),
+            );
+
+            const result4 = await client1.lolwut(6, undefined);
+            expect(intoString(result4)).toEqual(
+                expect.stringContaining("Redis ver. "),
+            );
+
+            const result5 = await client1.lolwut(5, [30, 4, 4]);
+            expect(intoString(result5)).toEqual(
+                expect.stringContaining("Redis ver. "),
+            );
+
+            // transaction tests
+            const transaction = new Transaction();
```

Comment:
Does it work to test lolwut transaction functionality in the usual spot (TestUtilities.ts)? Java does not have a separate transaction test for standalone/cluster, both are tested in TransactionTestUtilities.java. If not and we have to keep the transaction test here, I wonder if we should do the transaction test in a separate test since it seems like we usually separate transaction tests. But if we decide to stick with both in one test, we can probably use the same client for the regular functionality and for the transactions, I don't think we need to create two clients like we do here

Reply:
You brought up two issues here.
In Python, if I recall correctly, it has its own test for transaction. 
In this file, I do see other developer combined UT and transaction tests together using two separated clients. 
I think in the end, we reach the purpose of testing both UT and transaction. 
However, if you think I need to split this test to two tests, I can do that too.

Reply:
As far as I can tell the only test that has two separate clients and has combined transaction and non-transaction calls is the WATCH test, and it does that because it is testing that one client that executed WATCH will abort the transaction if another client changes the value of one of the keys that it is watching. I guess its not a big deal if we test both functionalities in the same test, but we don't need two separate clients - the WATCH test only needed two clients for the reasons I mentioned earlier. Can you remove the second client and execute the test with just the one?